pandas/_libs/tslibs/parsing.pyx

```diff
@@ -6,8 +6,10 @@ import sys
 import re
 import time
 
-from cpython.datetime cimport datetime
+from libc.string cimport strchr
+from cpython.datetime cimport datetime, PyDateTime_IMPORT, PyDateTimeAPI
 
+PyDateTime_IMPORT
```

Comment:
nitpick: another line between the libc import and the cpython.datetime import, then remove line 11.  (this should match the pattern in other _libs modules that import these)

Reply:
We have removed this line altogether, as it's not needed after switch to `datetime_new()`.